src/icecast-metadata-player/types/icecast-metadata-player.d.ts

```diff
@@ -0,0 +1,205 @@
+declare module "icecast-metadata-player" {
+  /** Icecast/Shoutcast ICY metadata values */
+  export interface IcyMetadata {
+    StreamTitle?: string; // ICY metadata updates are sometimes just an empty string, which will result in an empty object
```

Comment:
I think it will look a little more tidy if we put comments/documentation above the property and wrap comments/documentation to 80-120 characters (and use multi-line comment format if the line is wrapped.) JSDoc style, I guess.